unicodetools/data/ucd/dev/DerivedAge.txt

```diff
@@ -2116,4 +2116,14 @@ FDC8..FDCE    ; 17.0 #   [7] ARABIC LIGATURE RAHIMAHU ALLAAH TAAALAA..ARABIC LIG
 
 # Total code points: 4803
 
+# ================================================
+
+# Age=V18_0
+
+# Newly assigned in Unicode 18.0.0 (September, 2026)
+
+20C3          ; 18.0 #       UAE DIRHAM SIGN
```

Comment:
DerivedAge (like the Derived files generally) is never hand-edited, only regenerated as described in https://github.com/unicode-org/unicodetools/blob/main/docs/pipeline.md#regenerate-ucd.

This means that it will falsely say that this file is targeted for 17.0 (because we don’t have an 18.0 yet). That is OK, it will get regenerated to 18 once we have an 18 and the merge/regenerate commands get run on that PR.

The CI is supposed to put up a warning saying that this does not actually target 17 (otherwise some people in the public get false hopes looking at the diffs), but it does not know about 18, only about provisional assignment. I will fix the CI.